### ISSUE TITLE
`nr dev` scripts do not run concurrently on Windows

### ISSUE THREAD

**Issue by zanfee** [antfu/eslint-config#284]
### Describe the bug

`nr dev` runs tsup in watch mode and eslint-flat-config-viewer. Both commands run indefinitely and therefore need to be executed concurrently. I assume that this is achieved by the & sign on Unix-based systems, however on my Windows system only the first command is executed.

There are multiple tools which can be used to run these scripts concurrently, namely [npm-run-all](https://www.npmjs.com/package/npm-run-all) and [concurrently](https://www.npmjs.com/package/concurrently). I don't have a preference here but settled with npm-run-all for my own fork (https://github.com/antfu/eslint-config/commit/0bb5146f3c122e3f6e94d066137017ee4430738d) because the command looked slightly cleaner.

I haven't created a PR yet because #277 changes the npm scripts and this would cause a merge conflict. But I would be happy to create one once this is merged if you agree to use either npm-run-all or concurrently.

### Reproduction

Here is a copy of the repo at the time this issue exists, I don't think this need to be more minimal: https://github.com/zanfee/eslint-config/tree/bug/nr-dev-windows

### System Info

```Shell
System:
  OS: Windows 10 10.0.22621
  CPU: (12) x64 AMD Ryzen 5 7600 6-Core Processor
  Memory: 16.56 GB / 31.16 GB
Binaries:
  Node: 18.17.1 - C:\Program Files\nodejs\node.EXE
  npm: 9.7.2 - C:\Program Files\nodejs\npm.CMD
  pnpm: 8.9.0 - ~\AppData\Local\pnpm\pnpm.CMD
Browsers:
  Edge: Chromium (117.0.2045.60)
  Internet Explorer: 11.0.22621.1
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by antfu** [comment]
I wonder if this would work https://petermekhaeil.com/til/pnpm-shell-emulator/ ?

**Comment by zanfee** [comment]
Good idea, the scripts seem to work just fine with shell-emulator. However when stopping it with `ctrl + c`, the terminal seems to crash and does not respond anymore. I will investigate this further later.

**Comment by zanfee** [comment]
> Good idea, the scripts seem to work just fine with shell-emulator. However when stopping it with `ctrl + c`, the terminal seems to crash and does not respond anymore. I will investigate this further later.

This problem did not occur on my laptop. I will do more testing/research in the coming days.

```
System:
  OS: Windows 10 10.0.22621
  CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
  Memory: 1.75 GB / 15.73 GB
Binaries:
  Node: 18.18.0 - C:\Program Files\nodejs\node.EXE
  npm: 10.2.0 - C:\Program Files\nodejs\npm.CMD
  pnpm: 8.9.0 - ~\AppData\Local\pnpm\pnpm.EXE
Browsers:
  Edge: Spartan (44.22621.2283.0), Chromium (117.0.2045.60), ChromiumDev (119.0.2151.2)
  Internet Explorer: 11.0.22621.1
```

**Comment by zanfee** [comment]
On the system I mentioned in my first post (https://github.com/antfu/eslint-config/issues/284#issue-1941795866) this problem occurs in Git Bash and in Windows CMD but not in WSL. When I stop the terminal command, the tsup command doesn't properly end (I'm not sure if this is related to tsup though, only noticed that eslint-viewer is unreachable immediately after terminating it).

Sometimes the terminal will freeze completely, sometimes I can still enter things but it wont do anything (until tsup actually stops, like you can see in the screenshot).

![image](https://github.com/antfu/eslint-config/assets/22038857/ad55fece-4db0-44bc-918c-59e3acffea07)

**Comment by antfu** [comment]
Merged #287 for now to move forward. I am not sure if the other issue you mentioned is related to this project or not (looks more like a WSL/setup issue to me). If any other thing we could do on our side, feel free to bring this back. Thanks.
